auth0/src/main/java/com/auth0/android/provider/OAuthManager.java

```diff
@@ -248,15 +252,17 @@ private Uri buildAuthorizeUri() {
     }
 
     private void addPKCEParameters(Map<String, String> parameters, String redirectUri) {
-        try {
-            createPKCE(redirectUri);
-            String codeChallenge = pkce.getCodeChallenge();
-            parameters.put(KEY_CODE_CHALLENGE, codeChallenge);
-            parameters.put(KEY_CODE_CHALLENGE_METHOD, METHOD_SHA_256);
-            Log.v(TAG, "Using PKCE authentication flow");
-        } catch (IllegalStateException e) {
-            Log.e(TAG, "Some algorithms aren't available on this device and PKCE can't be used for authentication, which is required.", e);
+        // If PKCE is not available on the device (should never  happen), just return here and
```

Comment:
Let's keep the check-in a single place. Given that OAuthManager is only used by the WebAuthProvider class (for login and logout), let's remove this from here and keep it in the login scenario outside.